usaspending_api/api_contracts/contracts/v2/search/spending_by_award_count.md

```diff
@@ -70,6 +75,10 @@ This endpoint takes award filters, and returns the number of awards in each awar
 + `object_class` (optional, array[string])
 + `program_activity` (optional, array[number])
 
+### NAICSCodesObject (object)
++ `included`: `11` array[string]
++ `excluded`: array[string]
+
```

Comment:
This will conflict with the changes in https://github.com/fedspendingtransparency/usaspending-api/pull/2336, and shows a different API  contract  from what will be served